src/lib/checkbox/checkbox.scss

```diff
@@ -240,6 +240,7 @@ md-checkbox {
   vertical-align: middle;
   white-space: nowrap;
   width: $md-checkbox-size;
+  flex: 0 0 $md-checkbox-size;
```

Comment:
Why not just applying `flex-shrink: 0`? 
Setting another flex basis should be unnecessary.